scripts/tcl_commands/floorplan.tcl

```diff
@@ -69,11 +69,11 @@ proc init_floorplan {args} {
 			$core_height <= [expr {$::env(FP_PDN_HOFFSET) + $::env(FP_PDN_HPITCH)}]} {
 				puts_warn "Current core area is too small for a power grid. The power grid will be minimized."
 
-			set ::env(FP_PDN_VOFFSET) [expr {$core_width/6.0}]
-			set ::env(FP_PDN_HOFFSET) [expr {$core_height/6.0}]
+			set ::env(FP_PDN_VOFFSET) [expr {double(round(100*($core_width)/6.0))/100}]
+			set ::env(FP_PDN_HOFFSET) [expr {double(round(100*($core_height)/6.0))/100}]
```

Comment:
I would probably be better to look at the manufacturing grid and use that in case it's something different.

Reply:
* My feeling was that is simply a rounding error from very 28.826666... to improper (```28.8270``` / 3 dec) instead of (```28.8300``` / 2 dec), so rounding to expected 2 dec now works.

Let me know if still unacceptable.

Reply:
@arlpetergadfort ,

Double checked:

* Another message: ```Pitch 30.5130 is too small for, must be atleast 30.5200```
* Looks like really is a mismatch in expectation of 2-digits vs 3-digits.
* I simply forced in .tcl a rounding to two digit (and does the trick), no other magic is in the patch.

Reply:
@arlpetergadfort 

* updated to a more elegant way of formatting to 2 decimal.

Let me know if still unacceptable.

Reply:
It doesn't make sense to something coarser than the manufacturing grid otherwise you would disallow legal inputs.

Reply:
Personally I would prefer it be an error rather than rounding as the user is clearly doing something wrong with an off-grid value.  Guessing how to fix is dangerous.

Reply:
* Removed (1) about the "rounding of division" that lead to part, lets get only PDN related behavior fixed for now.

Reply:
@tdene and I were wondering if you will open another PR with the fix for `floorplan.tcl`: https://github.com/The-OpenROAD-Project/OpenLane/pull/1038/files/5b1d54789d3a17c66ba73964e9d7961d2decfacb#diff-3b1e9d4927839bb0fe38be8ece41a6d968939bc388a109515846167b96c9ad4d